custom/icds_reports/ucr/data_sources/dashboard/pregnant_tasks.json

```diff
@@ -32,7 +32,7 @@
             "type": "property_name",
             "property_name": "tasks_type"
           },
-          "property_value": "pregnant"
+          "property_value": "pregnancy"
```

Comment:
was wondering why this data source wasn't populating. this was correct in the spec, but wrong in an email :/